src/commands/translation/pull.ts

```diff
@@ -41,6 +41,11 @@ export default class TranslationPull extends BaseCommand<
     force: Flags.boolean({
       summary: "Remove the confirmation prompt.",
     }),
+    format: Flags.option({
```

Comment:
I guess this works too, but i think we may be defining a custom flag here that is the same thing as the string flag like `Flags.string({ options: ["json", "po"], default: "json" })`? If the latter works, then I'd prefer to use the string flag.

Reply:
There’s no way to type the string flag, so if we stick with the string flag, we’d need to cast in a few places.  Looking at the docs, this seems like the recommended approach for an enum.

Reply:
Ahh I got you, i see what you mean. This sounds like the right solution then, thanks for explaining!

Reply:
No problem! Spent a bunch of time trying with the string, but it doesn't accept a type argument (like the custom flag does).